keps/sig-contributor-experience/0007-20180403-community-forum.md

```diff
@@ -0,0 +1,197 @@
+- Lack of developer participation
+  - The mailing lists work, how suitable is Discourse to replace a mailing list these days? CNCF has tried Discourse in the past. See [@cra's post](https://twitter.com/cra/status/981548716405547008)
+  - [Discussion on the pros and cons of each](https://meta.discourse.org/t/discourse-vs-email-mailing-lists/54298)
+  - We have enough churn and new Working Groups that we could pilot a few, opt-in for SIGs that want to try it? 
+- A community forum is asynchronous, whereas chat is realtime.
+  - This doesn't solve our Slack lock-in concerns, but can be a good first step in being more active in running our own community properties so that we can build out own own resources. 
+  - Ghost have [totally migrated to Discourse](https://twitter.com/johnonolan/status/980872508395188224?s=12) and shut down their Slack.
+    - We should keep an eye on this and see what data we can gleam from this. Engage with Ghost community folks to see what lessons they've learned.
+    - Not sure if getting rid of realtime chat entirely is a good idea either. 
+- [GDPR Compliance](https://www.eugdpr.org/)
+  - Lots of data retention options in Discourse. 
+  - We'd need to engage with upstream on their plans for this, we would want to avoid having to manage this ourselves. 
+
+#### References from other projects
+
+- [Chef RFC](https://github.com/chef/chef-rfc/blob/master/rfc028-mailing-list-migration.md)
+  - [Blog post](https://coderanger.net/chef-mailing-list/) from a community member - good mailing list and community feedback here. 
+- [Swift's Plan](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20170206/031657.html) - Long discussion, worth reading
+- [HTM Forum](https://discourse.numenta.org/t/guidelines-for-using-discourse-via-email/314)
+- [Julia](https://discourse.julialang.org/t/discourse-as-a-mailing-list/57) - It might be useful for us to investigate pregenerating the mail addresses?
+- [How's Discourse working out for Ghost](https://forum.ghost.org/t/hows-discourse-working-out-for-ghost/947) - We asked them for some direct feedback on their progress so far 
+
+
+## Graduation Criteria
+
+There will be a feedback subforum where users can directly give us feedback on what they'd like to see. Metrics and site usage should determine if this will be viable in the long term.
+
+After a _three month_ prototyping period SIG Contributor Experience will:
+
+- Determine if this is a better solution than what we have, and figure out where this would fit in the ecosystem
+  - There is a strong desire that this would replace an existing support venue, SIG Contributor Experience will weigh the options.  
+- If this solution is not better than what we have, and we don't want to support yet another tool we we would shut the project down.
+- If we don't have enough information to draw a conclusion, we may decide to extend the evaluation period.
```

Comment:
Having written policies for moderation, adding topics, etc. should be part of this graduation criteria somehow.

Reply:
I was thinking since we have to have moderation policies for the mailing lists anyway (which we haven't written yet) that it would make sense to have one set of moderation policies across the project so we're consistent across the board. I'll make that more explicit next time I PR.